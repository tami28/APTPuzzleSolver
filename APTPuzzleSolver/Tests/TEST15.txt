NumElements=72
12 0 1 0 0
15 0 1 1 -1
8 -1 1 0 1
59 0 1 0 -1
55 0 -1 1 -1
72 -1 0 -1 -1
35 1 0 1 -1
13 -1 -1 1 1
54 -1 -1 0 -1
33 0 0 1 -1
20 -1 1 1 -1
3 -1 0 0 -1
9 0 1 -1 -1
36 1 1 0 1
19 0 -1 0 0
42 0 -1 -1 0
17 1 -1 1 1
71 -1 -1 0 1
63 0 0 -1 0
57 1 0 1 -1
28 -1 -1 0 0
31 0 1 -1 -1
30 1 1 1 -1
21 -1 0 1 1
4 -1 -1 -1 1
32 1 0 0 1
24 0 -1 0 1
26 0 1 -1 0
11 1 0 0 0
16 0 -1 0 1
56 0 1 1 -1
69 -1 -1 0 -1
27 0 -1 1 0
64 -1 1 0 1
6 0 -1 -1 0
40 1 -1 0 1
70 0 0 1 -1
68 -1 0 1 0
22 -1 0 0 1
38 0 0 1 -1
37 -1 0 -1 1
67 1 0 -1 1
25 1 0 1 -1
39 -1 0 1 1
44 -1 0 0 1
41 0 0 0 -1
7 0 1 0 0
10 0 -1 -1 0
51 1 1 1 1
52 -1 1 0 1
66 0 1 -1 -1
5 1 1 1 0
46 -1 -1 0 -1
2 0 1 0 0
1 0 -1 -1 0
18 1 1 1 0
43 -1 0 1 0
53 -1 0 1 0
47 -1 0 0 0
48 0 -1 0 0
34 0 0 1 0
23 -1 0 -1 0
60 1 -1 0 0
45 0 1 0 1
50 0 0 -1 -1
14 1 0 -1 0
49 1 -1 -1 0
62 1 -1 1 0
29 -1 1 -1 1
65 1 0 -1 0
58 1 1 1 0
61 -1 0 0 1