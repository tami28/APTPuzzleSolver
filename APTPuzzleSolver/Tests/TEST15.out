70, 68, 22, 38, 37, 67, 25, 39
44, 26, 11, 16, 56, 69, 27, 64
6, 40, 63, 57, 28, 31, 30, 21
4, 32, 24, 33, 20, 3, 9, 36
19, 42, 17, 71, 12, 15, 8, 59
55, 72, 35, 13, 54, 41, 7, 10
51, 52, 66, 5, 46, 2, 45, 50
14, 49, 62, 29, 65, 58, 61, 1
18, 43, 53, 47, 48, 34, 23, 60